pkgs/applications/version-management/gitea/default.nix

```
@@ -33,6 +34,16 @@ buildGoPackage rec {
 
   patches = [
     ./static-root-path.patch
+    # CVE-2022-0905
+    (fetchpatch {
+      url = "https://github.com/go-gitea/gitea/commit/1314f38b59748397b3429fb9bc9f9d6bac85d2f2.patch";
```

```suggestion
      name = "CVE-2022-0905.patch"
      url = "https://github.com/go-gitea/gitea/commit/1314f38b59748397b3429fb9bc9f9d6bac85d2f2.patch";
```

This will allow some automated tools to detect that this issue has been fixed in this package.

---

@risicle: Nice, I didn't know that.